herddb-core/src/main/java/herddb/storage/DataStorageManager.java

```diff
@@ -64,6 +64,10 @@
     public abstract List<Record> readPage(String tableSpace, String uuid, Long pageId)
         throws DataStorageManagerException, DataPageDoesNotExistException;
 
+    public abstract void ensureIndexDirectory(String tableSpace, String indexName) throws DataStorageManagerException;
+    
+    public abstract void ensureTableDirectory(String tableSpace, String uuid) throws DataStorageManagerException;
```

Comment:
DataStorageManager is some more "generic"... shouldn't be better a initTable, bootTable or prepareTable?

Reply:
renamed to "bootTable"